pipeline.py

```diff
@@ -0,0 +1,218 @@
+
+          drive_client_wrapper.init_client_from_info(credentials_info)
+
+     # Load phone number <-> UUID table
+     print("Loading Phone Number <-> UUID Table...")
+     with open(phone_number_uuid_table_path, "r") as f:
+          phone_number_uuid_table = PhoneNumberUuidTable.load(f)
+     
+     # Load messages
+     messages_datasets = []
+     for i, path in enumerate(message_paths):
+          print("Loading Episode {}/{}...".format(i + 1, len(message_paths)))
+          with open(path, "r") as f:
+               messages_datasets.append(TracedDataJsonIO.import_json_to_traced_data_iterable(f))
+
+     # Load surveys
+     print("Loading Demographics ...")
+     with open(s01_demog_input_path, "r") as f:
+          s01_demographics = TracedDataJsonIO.import_json_to_traced_data_iterable(f)
+     
+     print("Loading Follow up surveys ...")
+     with open(s01_follow_up_survey_input_path, "r") as f:
+          s01_follow_up_survey = TracedDataJsonIO.import_json_to_traced_data_iterable(f)
+     
+     # Add survey data to the messages
+     print("Combining Datasets...")
+     data = CombineRawDatasets.combine_raw_datasets(user, messages_datasets, [s01_demographics, s01_follow_up_survey])
+
+     print("Translating Rapid Pro Keys...")
+     data = TranslateRapidProKeys.translate_rapid_pro_keys(user, data, pipeline_configuration, prev_coded_dir_path)
+
+     print("Auto Coding Messages...")
+     data = AutoCodeShowMessages.auto_code_show_messages(user, data, icr_output_dir, coded_dir_path)
+
+     print("Exporting production CSV...")
+     data = ProductionFile.generate(data, production_csv_output_path)
+
+     print("Auto Coding Surveys...")
+     data = AutoCodeSurveys.auto_code_surveys(user, data, phone_number_uuid_table, coded_dir_path)
+     
+     print("Writing TracedData to file...")
+     IOUtils.ensure_dirs_exist_for_file(json_output_path)
+     with open(json_output_path, "w") as f:
+          TracedDataJsonIO.export_traced_data_iterable_to_json(data, f, pretty_print=True)
+     
+     # Upload to Google Drive, if requested.
+     # Note: This should happen as late as possible in order to reduce the risk of the remainder of the pipeline failing
+     # after a Drive upload has occurred. Failures could result in inconsistent outputs or outputs with no
+     # traced data log.
+     if pipeline_configuration.drive_upload is not None:
+          print("Uploading CSVs to Google Drive...")
+
+          production_csv_drive_dir = os.path.dirname(pipeline_configuration.drive_upload.production_upload_path)
+          production_csv_drive_file_name = os.path.basename(pipeline_configuration.drive_upload.production_upload_path)
+          drive_client_wrapper.update_or_create(production_csv_output_path, production_csv_drive_dir,
+                                                  target_file_name=production_csv_drive_file_name,
+                                                  target_folder_is_shared_with_me=True)
+          '''
```

Comment:
If you're committing commented code you should add a TODO explaining why. 

It's super obvious to me in this case, but it also helps to remind you to uncomment it later.

Reply:
Nice tip thanks!